SimpleBrowser/SimpleBrowser.csproj

```diff
@@ -1,139 +1,49 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{041EB5E9-DE14-41BA-B59D-F77612578CD6}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>SimpleBrowser</RootNamespace>
-    <AssemblyName>SimpleBrowser</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
-    <RestorePackages>true</RestorePackages>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>TRACE;DEBUG</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <Description>SimpleBrowser is a lightweight, yet highly capable browser automation engine designed for automation and testing scenarios.</Description>
+    <Copyright></Copyright>
+    <AssemblyTitle>SimpleBrowser</AssemblyTitle>
+    <NeutralLanguage>en-US</NeutralLanguage>
+    <VersionPrefix>0.5.79</VersionPrefix>
+    <Authors>Nathan Ridley, Teun Duynstee, Kevin Yochum, Joe Feser</Authors>
+    <TargetFrameworks>net40</TargetFrameworks>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>

```

Comment:
It looks like the new project file format may have changed the way TreatWarningsAsErrors works. When I opened the solution in VS2017, the "Treat Warnings As Errors" setting for the SimpleBrowser project was set to "Specific Warnings" and set to ignore "NU1605". I changed this setting to "All". When I did that, the following was added to the project file:
```xml
<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net40|AnyCPU'">
    <WarningsAsErrors />
</PropertyGroup>
```
We'll need this setting in the new project file - treating all warnings as errors.

Reply:
This modification would be needed in the unit test and sample application project files as well.

Reply:
Added the item to all csprojs, without condition of configuration (release/debug).